### PR TITLE
build prod version

### PR DESCRIPTION
**Pull request by sebastianovide**
I've tried to reenable CD using the old travis, but hey have changed the pricing model and we've run out of quota.
Trying to move it all to Github actions.

This is just building the prod react (copied some from the existing "deploy-to-dev"). 

The next steps would be to build the cordova, sign it, and deploy to testflight.

### CODE REVIEW

**Review by joykuotw — APPROVED** [review]

File: scripts/runWithVars.sh [hunk 1]
```diff
@@ -11,7 +11,8 @@ cp node_modules/first-input-delay/dist/first-input-delay.min.js  public/
 VERSION_MAYOR=$(echo $npm_package_version| cut -d'.' -f 1)
 VERSION_MINOR=$(echo $npm_package_version| cut -d'.' -f 2)
 
-export REACT_APP_BUILD_NUMBER=${TRAVIS_BUILD_NUMBER:-"0"}
+BUILD_NUMBER=${TRAVIS_BUILD_NUMBER:-$GITHUB_RUN_NUMBER}
```
Comment (joykuotw) [inline scripts/runWithVars.sh:14]: If we decide to not use travis, then the TRAVIS_BUILD_NUMBER is not needed. We can clean it up now or later.

File: .github/workflows/ios-testflight.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+
+jobs:
+  ios:
+    runs-on: macos-latest
+    # fetch the code
+    steps:
+      - uses: actions/checkout@v2
+
+# TODO: auto bump version: need to make it happen only once (It is happening also in deploy-to-dev.yml)
```
Comment (munroTom) [inline .github/workflows/ios-testflight.yml:16]: Tbh we could probably delete the deploy to dev one if we're doing this - or just add it as an extra step in this action?

Comment (joykuotw) [reply .github/workflows/ios-testflight.yml:16]: Sure, we can use the same workflow with multiple jobs. Have to rename the name of the workflow and the yml file first.

**Comment by neilfulwiler** [comment]
This looks great!

**Pull request merged**